app/main.py

```diff
@@ -1,8 +1,45 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(self, comfort_class: int,
+                 clean_mark: int, brand: str) -> None:
```

Comment:
Ajuste a identação desse método para deixá-lo mais legível. Deixe sempre a abertura e fechamento dos parênteses em linhas separadas dos parâmetros, caso sejam muitos parâmetros.

```suggestion
    def __init__(
        self,
        comfort_class: int,
        clean_mark: int,
        brand: str
    ) -> None:
```